src/layouts/SiteDashboard/components/ReviewRequestCard.tsx

```diff
@@ -36,8 +36,7 @@ export const ReviewRequestCard = ({
       }
     >
       {reviewRequest.status === ReviewRequestStatus.PENDING &&
-        // TODO: To check the logged in user's email address instead
-        reviewRequest.author !== userId && (
+        reviewRequest.author !== email && (
```

Comment:
do we need to also update the `reviewRequest.author` type here?

Reply:
Hmm the types do match currently, both being `string`.

Reply:
as in because now we are using `email` and not `userId`, is this check still logically sound?

Reply:
Yup, it was originally intended to be checking based on the email. The userId was just a placeholder.